apps/widhrt2/widget.js

```diff
@@ -0,0 +1,49 @@
+(function(){
+  if (!Bangle.isHRMOn) return; // old firmware
+  var hp = Bangle.setHRMPower;
+  Bangle.setHRMPower = () => {
+    hp.apply(Bangle, arguments);
+    WIDGETS.widhrt.draw();
```

Comment:
Should we preserve the return value?

```suggestion
    var r = hp.apply(Bangle, arguments);
    WIDGETS.widhrt.draw();
    return r;
```

Reply:
Happy to go with whatever you guys decide.
I only did it this way as that code became part of widhrt at somepoint and just copied it from there.


Just out of interest how are pull requests working these days ?
I am more than happy for my code to be reviewed and glad that we have this in place, it can only lead
to better apps and code.
I might have missed some announcements though.
Bob are you working for Gordon or were you volunteered etc ?
Is there a team of reviewers etc ?

Reply:
Ah gotya, in that case I'll resolve these comments and perhaps make these adjustments in a separate PR later - thanks for letting me know!

And yeah, Gordon asked if anyone would mind sharing the load (in #2993), what with holidays and so on. Just volunteering :)